### ISSUE TITLE
KeyError: 'regularMarketPrice' in version 0.1.49 

### ISSUE THREAD

**Issue by ricardomerces** [ranaroussi/yfinance#124]
Dear Ran,

```
bitcoin = yf.Ticker("BTC-USD")
print(bitcoin.info['regularMarketPrice'])
```

Work fine in 0.1.45

**Comment by GregoryMorse** [comment]
This release is problematic.  Also note this one:
```
  File "C:\ProgramData\Anaconda3\lib\site-packages\yfinance\base.py", line 279, in _get_fundamentals
    self._info = data['summaryProfile']

KeyError: 'summaryProfile'
```

It could be because yahoo seems to be doing intensive updates to their backend and frontend of the site.  There is no good solution except to try to keep up with the changes and the maintainer continue making releases which deal with this.  These also is likely because the testing is not enough.

I ask the maintainer to make sure to test the following 4 very important test cases:
index e.g. S&P500 '^GSPC'
mutual fund e.g. Vanguard 500 Index Fund 'VFINX'
ETF e.g. Russell 2000 Growth 'IWO'
security e.g. Microsoft 'MSFT' (probably what is being tested currently)

These things all worked in the prior release to the current one.  More logic needed as obviously the summaryProfile is not available in an index.  The idea is to return as much info as possible like the old functionality.

**Comment by ranaroussi** [comment]
Fixed in `0.1.50`. Please re-open if the problem persists.
